### PR TITLE
Openbabel with python.

### PR DESCRIPTION
**Pull request by cwant**
Could use a review of this.

### CODE REVIEW

**Comment by mboisson** [comment]
Tests need to be disabled ?

File: easybuild/easyconfigs/o/OpenBabel/OpenBabel-3.0.0-gompi-2018.3-Python.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name = 'OpenBabel'
+version = '3.0.0'
+versionsuffix = '-Python-%(pyver)s'
```
Comment (mboisson) [inline easybuild/easyconfigs/o/OpenBabel/OpenBabel-3.0.0-gompi-2018.3-Python.eb:3]: remove versionsuffix, we don't typically use it.

File: easybuild/easyconfigs/o/OpenBabel/OpenBabel-3.0.0-gompi-2018.3-Python.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    ('Eigen', '3.3.5'),
+    ('Boost', '1.68.0'),  # optional: for Maestro formats
+]
+
+configopts = ' -DCMAKE_SKIP_RPATH=ON '
+configopts += '-DBoost_INCLUDE_DIR=$EBROOTBOOST/include -DBoost_LIBRARY_DIR_RELEASE=$EBROOTBOOST/lib '
+# Enable support for OpenMP compilation of forcefield code (optional)
+configopts += '-DENABLE_OPENMP=ON '
+
+runtest = False
```
Comment (mboisson) [inline easybuild/easyconfigs/o/OpenBabel/OpenBabel-3.0.0-gompi-2018.3-Python.eb:33]: Tests need to be disabled ?

File: easybuild/easyconfigs/o/OpenBabel/OpenBabel-3.0.0-gompi-2018.3-Python.eb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+sources = ['%%(namelower)s-%s.tar.gz' % version.replace('.', '-')]
+checksums = ['5c630c4145abae9bb4ab6c56a940985acb6dadf3a8c3a8073d750512c0220f30']
+
+builddependencies = [
+    ('CMake', '3.16.3'),
+    ('SWIG', '3.0.12'),
+]
+
+dependencies = [
+    ('Python', '3.7.4'),
```
Comment (mboisson) [inline easybuild/easyconfigs/o/OpenBabel/OpenBabel-3.0.0-gompi-2018.3-Python.eb:23]: Can it be compiled with `multi_deps` ?

File: easybuild/easyconfigs/o/OpenBabel/OpenBabel-3.0.0-gompi-2018.3-Python.eb [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+]
+
+configopts = ' -DCMAKE_SKIP_RPATH=ON '
+configopts += '-DBoost_INCLUDE_DIR=$EBROOTBOOST/include -DBoost_LIBRARY_DIR_RELEASE=$EBROOTBOOST/lib '
+# Enable support for OpenMP compilation of forcefield code (optional)
+configopts += '-DENABLE_OPENMP=ON '
+
+runtest = False
+
+postinstallcmds = [ "/cvmfs/soft.computecanada.ca/easybuild/bin/setrpaths.sh --path %(installdir)s --add_path %(installdir)s/lib" ]
```
Comment (mboisson) [inline easybuild/easyconfigs/o/OpenBabel/OpenBabel-3.0.0-gompi-2018.3-Python.eb:35]: Is this needed ?  It sometimes is, but usually, it should not.

**Comment by cwant** [comment]
Hi Maxime, thank you for the review. I have since solved this by editing the existing recipe to hook up to python support (with assistance from Ali and Bart). I should have closed the PR since it is obsolete.

**Pull request closed without merging**